library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1303,6 +1314,61 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       this.output.format(format);
     }
 
+    private static class BITMAPINFOHEADER {
```

Comment:
See comment on other PR about getting rid of unnecessary code. This whole thing is basically the same as skipping some bytes, checking a constant (biCompression) and skipping some more bytes. It's not necessary to define a class containing a whole bunch of stuff that's unused to do this.